recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
     - 4535-relax_protobuf_to_under_4.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
Can you please add a test to the `tests: commands:` section below that checks whether the new files are part of the package, e.g. using `test -f $PREFIX/…`

Reply:
as far as we can see, installation rules are added only for Linux. So, on Windows we still don't have cmake, headers, etc

Reply:
Use a platform selector? It is somewhat common for win/unix to have different artifacts.

Reply:
The patch needs to be improved to have aligned behavior for Win / Unix. 
Users should expect cmake interface to be available in all cases